tools.go

```diff
@@ -0,0 +1,16 @@
+// +build tools
+
+package main
+
+import (
+	_ "github.com/Songmu/gocredits"
+	_ "github.com/golang/mock/mockgen"
+	_ "github.com/jessevdk/go-assets-builder"
+	_ "github.com/jpillora/go-tcp-proxy"
```

Comment:
It looks that this breaks `make credits` because it lacks license file.  Should we somehow exclude license files of tools since they are not included in the generated binary?

Reply:
It's better to include in the CREDITS file even if it is a tool dependency. I just opened https://github.com/jpillora/go-tcp-proxy/issues/13 and wait for the information.

Reply:
I understand that tool dependencies should be listed but I don't think tool (or test library) licenses should be included because we are not redistributing them.  Including them might be even misleading to users.

Reply:
Okay, but we can't distinguish them from single go.mod file so maybe we can either

- create `_tools/go.mod` and always change the working directory to `_tools` on installing them in `Makefile`
- or change the working directory to a temporary directory (or just `cd &&` which I often use in my project) and give up locking the versions of tools dependencies.

Reply:
`_tools/go.mod` sounds good.